docs/src/pages/_document.tsx

```diff
@@ -1,9 +1,12 @@
+import { GoogleAnalytics } from "@next/third-parties/google";
 import { Head, Html, Main, NextScript } from "next/document";
 
 export default function Document() {
   return (
     <Html lang="en">
-      <Head />
+      <Head>
+        <GoogleAnalytics gaId="G-KWK6N3X1ZD" />
```

Comment:
@dimaMachina is this the recommended way to put google analytics on our site?

Reply:
Use nex.js recommendations